### PR TITLE
REL-2147 DOCKER-42 Update SonarQube to 9.3.0; officially support DCE Chart

### PR DESCRIPTION
**Pull request by wouter-admiraal-sonarsource**

### CODE REVIEW

File: charts/sonarqube/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 name: sonarqube
 description: SonarQube offers Code Quality and Code Security analysis for up to 27 languages. Find Bugs, Vulnerabilities, Security Hotspots and Code Smells throughout your workflow.
-version: 1.6.5
-appVersion: 9.2.4
+version: 1.6.6
```
Comment (tobias-trabelsi-sonarsource) [inline charts/sonarqube/Chart.yaml:4]: i don't think an application version bump is a patch release :sweat_smile: 
as there are a few changes in this release that are user facing, i think this even justifies a major version bump. wdyt?

**Review by tobias-trabelsi-sonarsource — CHANGES_REQUESTED** [review]
You need to change the `values.yaml` as well to reference the new images

File: charts/sonarqube/Chart.yaml (hunk 1 above)
Comment (wouter-admiraal-sonarsource) [reply charts/sonarqube/Chart.yaml:4]: Works for me 👍

**Review by tobias-trabelsi-sonarsource — APPROVED** [review]
LGTM :tada:

**Pull request merged**